kinto_http/tests/test_client.py

```diff
@@ -620,6 +620,61 @@ def test_update_collection_can_deduce_id_from_data(self):
             headers=None)
 
 
+class HistoryTest(unittest.TestCase):
+    def setUp(self):
+        self.session = mock.MagicMock()
+        self.client = Client(session=self.session)
+
+    def test_basic_retrivial_of_bucket_history(self):
+        mock_response(self.session)
+        self.client.get_history(bucket='mybucket')
+        url = '/buckets/mybucket/history'
+        self.session.request.assert_called_with('get', url, headers={}, params={})
+
+    def test_filter_sorting_operations_on_bucket_history(self):
+        mock_response(self.session)
+        self.client.get_history(bucket='mybucket',
+                                _limit=2,
+                                _sort='-last_modified',
+                                _since='1533762576015')
+
+        url = '/buckets/mybucket/history'
+        self.session.request.assert_called_with('get', url, headers={},
+                                                params={'_limit': 2,
+                                                        '_sort': '-last_modified',
+                                                        '_since': '1533762576015'}
+                                                )
+
+    def test_grouping_on_bucket_history(self):
+        mock_response(self.session)
+        self.client.get_history(bucket='mybucket', resource_name='collection')
+        url = '/buckets/mybucket/history'
+        self.session.request.assert_called_with('get', url, headers={},
+                                                params={'resource_name': 'collection'}
+                                                )
+
+    def test_purging_of_history(self):
+        mock_response(self.session)
+        self.client.purge_history(bucket='mybucket')
+        url = '/buckets/mybucket/history'
+        self.session.request.assert_called_with('delete', url, headers=None)
+
+    def test_get_record_by_revision_id(self):
+        mock_response(self.session)
+        self.session.request.return_value = ({'data': [{'target': {}}]}, mock.sentinel.count)
```

Comment:
@leplatrem Can you help me with this line? Is this acceptable? Actually i don't understand the way how mock.sentinel does work here...

Reply:
In general would it not be a good idea for the method update_record() to also return the revision id?

Reply:
Don't bother about the `mock.sentinel.count`. 

But put something realistic in the `target` object. And test that you obtain this as a response of `get_record()` below